index.html

```diff
@@ -1,299 +1,234 @@
+                            <li>GitLab</li>
+                            <li>Terminal</li>
+                            <li>WSL</li>
+                            <li>Remote Pair-Programming</li>
+                            <li>Teamwork</li>
+                            <li>Mentoring</li>
+                            <li>Code Reviews</li>
+                            <li>Mobile/Responsive Development</li>
+                        </ul>
+                    </li>
+                </ul>
+            </div>
         </div>
-        <div class="card katchy">
-          <div class="card-description">
-            <h2>Katchy</h2>
-            <p class="white-description">
-              This is a website for recipes made with Ruby on Rails
-            </p>
-            <ul class="project-stacks card-stacks">
-              <li>Ruby</li>
-              <li>Ruby on Rails</li>
-              <li>HTML</li>
-              <li>CSS</li>
-              <li>ERB</li>
-              <li>RSPec</li>
-            </ul>
-            <a
-              href="https://github.com/Spayco/katchy/"
-              class="card-link link"
-              target="_blank"
-              rel="noreferrer noopener"
-              >See project</a
-            >
-          </div>
-        </div>
-        
-      </div>
-      <div class="about-me">
-        <section class="about-me-section">
-          <h2>About me</h2>
-          <p class="about-me-text">
-            I am a skilled Full-Stack Web Developer who loves making pages with
-            beautiful UI. I have studied at Microverse, and I am experienced in
-            remote work.
-            <br />
-            I've also been a good helping hand to a lot of developers in
-            different communities!
-            <br />
-            if you'd like to get in contact about potential job opportunities,
-            please reach out!
-          </p>
-          <a
-            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
-            class="link"
-            target="_blank"
-            rel="noreferrer noopener"
-          >
-            Get my resume
-          </a>
-          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        <section class="contact-container">
+            <div class="contact-left">
+                <h3 class="contact-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+            </div>
+            <div class="contact-right">
+                <form action="https://formspree.io/f/xbjqgnpd" method="POST" class="contact-form">
+
+                    <input type="text" id="fullname" name="fullname" placeholder="Full name" required>
+                    <input type="email" id="email" name="email" placeholder="Email address" required>
+                    <textarea id="message" name="message" placeholder="How can i help you?" required></textarea>
```

Comment:
- [ ] **The valid maximum length of message input should be 500 characters.**

Currently, there is not maximum length for your message input field. It is a mandatory project requirement that message field should allow a maximum of 500 characters. Please fix this.

Hint: You can use an attribute that takes care of input validation related to length.